lib/aida/wit_ai.ex

```diff
@@ -138,6 +138,7 @@ defmodule Aida.WitAi do
       skill = bot.skills |> Enum.find(fn skill -> skill.id == skill_id end)
       %{confidence: confidence, skill: skill}
     end)
+    |> Enum.filter(fn %{skill: skill} -> Map.get(skill, :training_sentences) end)
```

Comment:
This seems like an impossible scenario, only skills with training sentences will be sent to wit.ai, so there is no point in filtering the others from the answer. Wit will never answer with a skill id that has no training sentences. The only way this could happen is if the update to wit.ai hasn't properly been done. If that happens, then we should consider discarding the entire answer and re-starting the bot all over again. Let's talk a little more about this.

Reply:
Ok, I'll take a deeper approach to this issue and I'll let you know. Thanks!